### ISSUE TITLE
dropbox link missing

### ISSUE THREAD

**Issue by NisargBhatt73** [heykeetae/Self-Attention-GAN#52]
Either of the download scripts don't work because supposedly the dropbox link for dataset doesn't work

On terminal
 **bash download.sh LSUN**

Error log-

Location: /s/raw/zt7d2hchrw7cp9p/church_outdoor_train_lmdb.zip [following]
--2020-10-23 00:49:32--  https://www.dropbox.com/s/raw/zt7d2hchrw7cp9p/church_outdoor_train_lmdb.zipReusing existing connection to www.dropbox.com:443.
HTTP request sent, awaiting response... 404 Not Found
2020-10-23 00:49:33 ERROR 404: Not Found.

Archive:  ./data/church_outdoor_train_lmdb.zip
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
unzip:  cannot find zipfile directory in one of ./data/church_outdoor_train_lmdb.zip or
        ./data/church_outdoor_train_lmdb.zip.zip, and cannot find ./data/church_outdoor_train_lmdb.zip.ZIP, period.

**Comment by chan4899** [comment]
Yes. Dropbox link is not working for both the datasets. For CelebA, you will have to download the dataset from a google drive link given at CelebA's website.
